packages/dev-middleware/src/types/Experiments.js

```diff
@@ -23,6 +23,12 @@ export type Experiments = $ReadOnly<{
    * interface.
    */
   enableOpenDebuggerRedirect: boolean,
+
+  /**
+   * Enables the new JS debugger network panel and network CDP events.
+   * When disabled, all CDP events and the network panel are disabled.
+   */
+  enableNetworkInspector: Boolean,
```

Comment:
This should be `boolean` for typechecking to pass.

```suggestion
  enableNetworkInspector: boolean,
```